src/test/java/org/folio/rest/impl/UsersAPIIT.java

```diff
@@ -233,7 +233,7 @@ void cannotCreateUserWithAddressButNoAddressType() {
       .build();
 
     usersClient.attemptToCreateUser(userToCreate)
-      .statusCode(is(400));
+      .statusCode(is(422));
```

Comment:
It is, I noted this in my comments above.  I can't remember if this is standard for fields that are required in the RAML, but it seems odd.  Wouldn't 400 be a more appropriate code?

Reply:
I did a bit of testing with various required fields in mod-users, they do indeed return 422 errors, with a message in the body detailing which field is missing.  The behavior is odd, but consistent.

Reply:
> It is, I noted this in my comments above. 

Apologies, I missed this.

> I can't remember if this is standard for fields that are required in the RAML, but it seems odd. Wouldn't 400 be a more appropriate code?

FOLIO doesn't have an explicit standard. I cannot recall the behaviour of PgUtil. I think it might prefer 422 and then fall back to 400.

Both are kinda appropriate, in the sense that FOLIO considers 422 a specialisation of 400 for a specific format of JSON response that describes a validation failure, rather than plain text.

Reply:
> I did a bit of testing with various required fields in mod-users, they do indeed return 422 errors, with a message in the body detailing which field is missing. The behavior is odd, but consistent.

I agree, it is a little odd. The decision to use 422 was made a long time ago and not really been reviewed since.